src/compilerlib/pb_parsing_parse_tree.ml

```diff
@@ -41,12 +41,12 @@ type message_field_label = [
 type oneof_field_label = unit 
 
 (** message field. 
-    
-   Note this field is parametrized with the label type 
-   so that it can be used both by normal field and one of 
-   field since the only difference between the 2 is 
-   the label.
- *)
+
```

Comment:
is there some standard formatting used in the repo?
I'm using `ocp-indent` and had many changes in files I've changed

Reply:
no formatting, ok to use ocp-indent.